src/ast/assignment-statement.ts

```diff
@@ -0,0 +1,18 @@
+import Context from '../semantics/context';
+import { AstNode } from '../type-definitions/plainscript-types';
+
+export default class AssignmentStatement extends AstNode<AssignmentStatement> {
+  constructor(public targets: any[], public sources: any[]) { super(); }
```

Comment:
Wouldn't it be
```
    constructor(public targets: VariableExpression[], sources: Expression[])
```

Reply:
Thank you for catching that ! Made a new `VariableExpression` type alias and corrected these any types !